ts/spi/provider_frame_test.ts

```diff
@@ -691,6 +695,12 @@ describe('ProviderFrame', () => {
             [elisaOtherDomainCred],
             elisaOtherDomainCred,
             msg.credentialResultMessage(requestId, redactedElisaCred))
+            .then(() => {
+              return localStateProvider.isAutoSignInEnabled(TEST_AUTH_DOMAIN);
+            })
+            .then((enabled) => {
+              expect(enabled).toBe(true);
+            })
             .then(done);
```

Comment:
nit: the done doesn't have to be in its own callback, you could move it in the function's body before.